doc/user/layouts/partials/toc.html

```diff
@@ -59,7 +59,7 @@ <h2 class="gradient_text dark">On this page</h2>
     const headings = $(".content h2, .content h3");
     let offsets = headings.toArray().map((h) => ({
       id: h.id,
-      offset: h.offsetTop - 75,  // 75 if banner is displayed, 65 otherwise
```

Comment:
I don't quite get this comment, but shouldn't this be "65 if banner is displayed"?

Reply:
Oh .. we're removing the banner from the docs site.  so that there is a bit more  bit more space between the top menu bar and the content (since the banner makes things squish down).  So, once it's gone, can squish on back up.